docs/getting-started/create.md

```diff
@@ -152,6 +152,7 @@ the ones your Magento backend doesn't support.
 
 - Remove `"@graphcommerce/[psp]"` from package.json
 - Remove all [psp] references from `pages/checkout/payment.tsx`
+- Currently these are: `[@graphcommerce/mollie-magento-payment, @graphcommerce/magento-payment-braintree]`
```

Comment:
Is addressed mentioned at line 150

Reply:
Wasn't too clear for me when reading it first, having the full strings available is helpful here.

Reply:
@FrankHarland suggested changes:

### Remove unused PSP's

The example has Payment Service Providers integrated. Remove
the ones your Magento backend doesn't support.

- Remove Payment Service integrations from package.json: `@graphcommerce/mollie-magento-payment` and/or `@graphcommerce/magento-payment-braintree`
- Remove Payment Service references from `pages/checkout/payment.tsx`